.gitlab-ci-darwin.yml

```diff
@@ -0,0 +1,83 @@
+variables:
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  GIT_SUBMODULE_STRATEGY: recursive
+  MODULE_CMAKE: "cmake/3.11.1"
+  MODULE_CLANG: "clang/8.0.1"
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
+  MODULE_MPI: "openmpi/p9/4.0.2-gcc_7.4.0"
+  NUM_MPI_PROC_TESTING: 2
+  OMP_NUM_THREADS: 40
+  CMAKE_BUILD_TYPE: "Release" 
+  Kokkos_ARCH_POWER9: "ON" 
+  Kokkos_ARCH_VOLTA70: "ON" 
+  Kokkos_ENABLE_CUDA: "ON" 
+  Kokkos_ENABLE_CUDA_UVM: "OFF"
+  Kokkos_ENABLE_CXX11: "ON" 
+  Kokkos_ENABLE_OPENMP: "ON" 
+  PARTHENON_DISABLE_HDF5: "OFF" 
+
+stages:
+  - performance-regression
+
+# Is performed before the scripts in the stages step
+before_script:
+  - env -i bash ./scripts/darwin/setup.sh
+  - env -i bash ./scripts/darwin/build_hdf5_parallel.sh
+  - export GOLD_STANDARD_VER=2
+  - if [ `cat tst/regression/gold_standard/current_version` != ${GOLD_STANDARD_VER} ]; then
+    echo "Updating regression Gold standard to ${GOLD_STANDARD_VER}." &&
+    wget -qO- https://pgrete.de/dl/parthenon_regression_gold_r${GOLD_STANDARD_VER}.tgz | tar -xz -C tst/regression/gold_standard; fi
+
+.gcc-mpi-cuda-performance-regression:
+  variables: 
+    BUILD_DIR: "build_power9_perf_regression_gcc_mpi"
+    CMAKE_CXX_COMPILER: $CI_PROJECT_DIR/external/Kokkos/bin/nvcc_wrapper
+  script:
+    - echo "./scripts/darwin/build.sh" > build_cmd.txt
+    - echo ${BUILD_DIR}              >> build_cmd.txt
+    - echo ${CMAKE_BUILD_TYPE}       >> build_cmd.txt
+    - echo ${CMAKE_CXX_COMPILER}     >> build_cmd.txt
+    - echo ${Kokkos_ARCH_POWER9}     >> build_cmd.txt
+    - echo ${Kokkos_ARCH_VOLTA70}    >> build_cmd.txt
+    - echo ${Kokkos_ENABLE_CUDA}     >> build_cmd.txt
+    - echo ${Kokkos_ENABLE_CUDA_UVM} >> build_cmd.txt
+    - echo ${Kokkos_ENABLE_CXX11}    >> build_cmd.txt
+    - echo ${Kokkos_ENABLE_OPENMP}   >> build_cmd.txt
+    - echo ${NUM_MPI_PROC_TESTING}   >> build_cmd.txt
+    - echo ${OMP_NUM_THREADS}        >> build_cmd.txt
+    - echo ${PARTHENON_DISABLE_HDF5} >> build_cmd.txt
+    - >
+      env -i bash ./scripts/darwin/build.sh
+      ${BUILD_DIR}
+      ${CMAKE_BUILD_TYPE}
+      ${CMAKE_CXX_COMPILER}
+      ${Kokkos_ARCH_POWER9}
+      ${Kokkos_ARCH_VOLTA70}
+      ${Kokkos_ENABLE_CUDA}
+      ${Kokkos_ENABLE_CUDA_UVM}
+      ${Kokkos_ENABLE_CXX11}
+      ${Kokkos_ENABLE_OPENMP}
+      ${NUM_MPI_PROC_TESTING}
+      ${OMP_NUM_THREADS}
+      ${PARTHENON_DISABLE_HDF5}
+  artifacts:
+    expire_in: 3 days
+    paths:
+      - ${CI_PROJECT_DIR}/${BUILD_DIR}/tst/regression/advection_convergence/advection-errors.dat
```

Comment:
Did you check of the performance plot is also generated?
https://github.com/lanl/parthenon/blob/c18c2267f6b0889d9a51bd304cb95f85bf80d438/.gitlab-ci-ias.yml#L119
We should keep this and push it to the PR (I'll see how that works from gitlab over the next days and report back)

Reply:
Yes it is generated, there was an error in the PATH though.

Reply:
Any updates?